src/components/Navbar/Navbar.js

```diff
@@ -7,6 +7,11 @@ import React, {
 } from 'react';
 import './navbar.css';
 
+import Lottie from 'lottie-react';
```

Comment:
Is Lottie planned to be used at a later time?

Reply:
don't think so. I have removed this line.